packages/jest-cli/src/TestRunner.js

```diff
@@ -379,6 +379,10 @@ class TestRunner {
   _setupReporters() {
     const config = this._config;
 
+    if (config.reporters) {
+      return this._addCustomReporters(config.reporters);
```

Comment:
@dmitriiabramov The configuration open `noDefaultReporters: true` has been added. One concern here is, Should this option only perform the desired behavior when it's used in tandem with `config.reporters` and otherwise give a warning maybe? Or should it work regardless of config.reporters?